Facing error in executing any cli command once OTEL_ENABLED flag is set to true

## Current Behavior
Facing error in executing any cli command once OTEL_ENABLED flag is set to true in the project.

## Expected Behavior
It should work properly without any error.

## Steps to Reproduce
1. Enable tracing in the project.
```
$ godspeed otel enable
       ,_,   ╔════════════════════════════════════╗
      (o,o)  ║        Welcome to Godspeed         ║
     ({___}) ║    World's First Meta Framework    ║
       " "   ╚════════════════════════════════════╝
otel installed successfully!
Observability has been enabled
```
2. It sets the flag in the `.env ` file of project. 
```
OTEL_ENABLED=true
```
3. Try to execute `godspeed` cli command again in the project.
```
$ godspeed 
{"Body":"OTEL_ENABLED is set, unable to initialize opentelemetry tracing.","Timestamp":"undefined000000","SeverityNumber":17,"SeverityText":"ERROR","Resource":{"service.name":"unknown_service:node"},"Attributes":{}}
{"Body":"Cannot find module '@godspeedsystems/tracing'\nRequire stack:\n- /home/gurjot/data/cli-test/godspeed-cli/node_modules/@godspeedsystems/core/dist/godspeed.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/utils/index.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/commands/create/index.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/index.js","Timestamp":"undefined000000","SeverityNumber":17,"SeverityText":"ERROR","Resource":{"service.name":"unknown_service:node"},"Attributes":{"err":{"type":"Error","message":"Cannot find module '@godspeedsystems/tracing'\nRequire stack:\n- /home/gurjot/data/cli-test/godspeed-cli/node_modules/@godspeedsystems/core/dist/godspeed.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/utils/index.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/commands/create/index.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/index.js","stack":"Error: Cannot find module '@godspeedsystems/tracing'\nRequire stack:\n- /home/gurjot/data/cli-test/godspeed-cli/node_modules/@godspeedsystems/core/dist/godspeed.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/utils/index.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/commands/create/index.js\n- /home/gurjot/data/cli-test/godspeed-cli/lib/index.js\n    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)\n    at Function.Module._load (node:internal/modules/cjs/loader:778:27)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)\n    at require (node:internal/modules/cjs/helpers:102:18)\n    at Object.<anonymous> (/home/gurjot/data/cli-test/godspeed-cli/node_modules/@godspeedsystems/core/dist/godspeed.js:184:9)\n    at Module._compile (node:internal/modules/cjs/loader:1103:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)\n    at require (node:internal/modules/cjs/helpers:102:18)\n    at Object.<anonymous> (/home/gurjot/data/cli-test/godspeed-cli/lib/utils/index.js:20:16)\n    at Module._compile (node:internal/modules/cjs/loader:1103:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)\n    at require (node:internal/modules/cjs/helpers:102:18)\n    at Object.<anonymous> (/home/gurjot/data/cli-test/godspeed-cli/lib/commands/create/index.js:15:17)\n    at Module._compile (node:internal/modules/cjs/loader:1103:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)\n    at require (node:internal/modules/cjs/helpers:102:18)\n    at Object.<anonymous> (/home/gurjot/data/cli-test/godspeed-cli/lib/index.js:34:33)\n    at Module._compile (node:internal/modules/cjs/loader:1103:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at node:internal/main/run_main_module:17:47","code":"MODULE_NOT_FOUND","requireStack":["/home/gurjot/data/cli-test/godspeed-cli/node_modules/@godspeedsystems/core/dist/godspeed.js","/home/gurjot/data/cli-test/godspeed-cli/lib/utils/index.js","/home/gurjot/data/cli-test/godspeed-cli/lib/commands/create/index.js","/home/gurjot/data/cli-test/godspeed-cli/lib/index.js"]}}}
```


## Screenshots or Screencast
![image](https://github.com/godspeedsystems/godspeed-cli/assets/88080236/95409be7-988c-4edf-893d-5c285d3864cc)

## Possible Solutions or Workarounds
Remove `@godspeedsystems/tracing` initialization from the core package and it should be initialized from the project itself.

--- a/src/godspeed.ts
+++ b/src/godspeed.ts
@@ -2,16 +2,6 @@
 import 'dotenv/config';
 import fs from 'fs';
 import { childLogger, initializeChildLogger, logger } from './logger';
-try {
-  if (process.env.OTEL_ENABLED == 'true') {
-    require('@godspeedsystems/tracing').initialize();
-  }
-} catch (error) {
-  logger.error("OTEL_ENABLED is set, unable to initialize opentelemetry tracing.");
-  logger.error(error);
-  process.exit(1);
-}
-
 var config = require('config');
 
 import { join } from 'path';

